engine/collection/synchronization/engine.go

```diff
@@ -370,7 +371,7 @@ func (e *Engine) pollHeight() {
 		Height: head.Height,
 	}
 	err = e.con.Multicast(req, synccore.DefaultPollNodes, e.participants.NodeIDs()...)
-	if err != nil {
+	if err != nil && !errors.Is(err, network.EmptyTargetList) {
```

Comment:
Removes logging noise from test logs when there is a single-node collector cluster.